frontend/src/components/common/types/Filters.ts

```diff
@@ -0,0 +1,9 @@
+export type Filters = {
```

Comment:
maybe rename this to LogFilters, since we'll also have filters for employees and residents in the future

Reply:
Yeah that makes sense, I will change it. 👍